lisp/ess-custom.el

```diff
@@ -2519,12 +2519,18 @@ default."
 ;; This will never need to be loaded independently of any of the other modules,
 ;; but they can all call it so we may as well put it here.
 
-(defcustom ess-help-pop-to-buffer t
-  "If non-nil ess-help buffers are given focus during the display.
-The default is t (except when `focus-follows-mouse' and
-`mouse-autoselect-window' are both t)."
+(define-obsolete-variable-alias 'ess-help-pop-to-buffer
+  'ess-help-pop-to-buffer-function "ESS 19.04")
+(defcustom ess-help-pop-to-buffer-function #'pop-to-buffer
```

Comment:
It's not needed, but it doesn't divert from the display-buffer framework (pop-to-buffer uses display-buffer). I thought maybe someone would like to add their own function, but perhaps it's not necessary.

Reply:
By "driven away" I meant that someone would need to add their own function instead of customizing  display-buffer-alist. It's a redundant customization in that sense.

Reply:
Ah, OK I think I see what you mean. I'll revert this part then.